scripts/heroku-run.bash

```diff
@@ -46,6 +46,7 @@ function putS3
 }
 
 java -ea \
+  -Xms2400m -Xmx2400m
```

Comment:
Does this value include what Heroku reports as "cache" as well as "rss"?  The sum of those is what their quota is based on.  From the logs, looks like our "cache" value tops out around 160MB.

Consider setting this a little lower, if we want to be sure?